server.dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0
+RUN mkdir /app
+WORKDIR /app
+COPY . .
+RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
+RUN apt-get install -y nodejs
+RUN apt update && apt install -y wget bash libatomic1 procps
+RUN wget -q "https://cdn.altv.mp/server/release/x64_linux/altv-server"
```

Comment:
```
RUN wget -q "https://cdn.altv.mp/server/release/x64_linux/altv-server"
RUN wget -q "https://cdn.altv.mp/js-module/release/x64_linux/modules/js-module/libjs-module.so" -P modules/js-module/
RUN wget -q "https://cdn.altv.mp/js-module/release/x64_linux/modules/js-module/libnode.so.102" -P modules/js-module/
```

These are all obtained by `npm run update`

```
chmod 777 altv-server
```

Should be moved to after `npm run update`.

Reply:
actually i didn't aware of this command sorry about that :D npm run update was a good command for this docker image man. :D i'll fix this up today.